app/models/user.rb

```diff
@@ -3,4 +3,17 @@
 class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  has_one_attached :avatar
+
+  validate :avatar_content_type, if: :was_attached?
```

Comment:
```suggestion
  validate :avatar_content_type, if: -> { avatar.attached? }
```

メソッド化するメリットがあまりないのと、「was = 過去の話？」という思ってしまいそうなのが紛らわしいので、ラムダに置き換えた方がいいと思います。

https://bootcamp.fjord.jp/pages/when-you-should-define-method

Reply:
他メソッドを呼び出すだけで、ご指摘の通り名前の意味も分かり辛いためラムダに置き換えます。